src/apiCalls.js

```diff
@@ -1,13 +1,11 @@
 function fetchAllMoviesData() {
-  return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies').then(
-    (response) => response.json()
-  );
+  return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    .then((response) => response.json())
```

Comment:
I noted this above, but we would like to see some error handling capability here. We can't assume that the server will always give a good response. Maybe the server is down at some point.